CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Liana daemon and GUI release notes
 
+## 0.4
+
+This fourth release brings support for descriptors with multiple recovery path as well as several
+usability improvements in the GUI around signing devices management, and more.
+
+### Features
+
+- We now support having multiple recovery path in a descriptor.
+- We now support more general descriptors: multisigs in the primary or any of the recovery paths
+  henceforth aren't required to use `multi()` anymore and the maximum number of keys per spending
+  path is thereby lifted.
+
+#### GUI-specific
+
+- You can now re-register the descriptor on a hardware signing device in the settings.
+- You can now change the alias of each of the signers from the settings panel.
+- At signing time we now warn you if the descriptor is not registered on the signing device.
+- The signer alias is now displayed along with its type when signing.
+- You can now connect to a running daemon without having to provide a path to its configuration,
+  provided it is using the default location for its data directory.
```

Comment:
Or that you provide the `--datadir` option when running the GUI. I love this new feature.